modules/ossm-troubleshooting-injection.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-injection_{context}"]
+= Troubleshooting sidecar injection
+
+{ProductName} does not automatically inject proxy sidecars to pods. You must opt in to sidecar injection.
+
+== Troubleshooting Istio sidecar injection
+
+Check to see if automatic injection is enabled in the Deployment for your application. If automatic injection for the Envoy proxy is enabled, there should be a `sidecar.istio.io/inject:"true"` annotation in the `Deployment` resource under `spec.template.metadata.annotations`.
+
+== Troubleshooting Jaeger agent sidecar injection
+
+Check to see if automatic injection is enabled in the Deployment for your application. If automatic injection for the Jaeger agent is enabled, there should be a `sidecar.jaegertracing.io/inject:"true"` annotation in the `Deployment` resource.
```

Comment:
```suggestion
Check to see if automatic injection is enabled in the `Deployment` resource for your application. If automatic injection for the Envoy proxy is enabled, there should be a `sidecar.istio.io/inject:"true"` annotation in the `Deployment` resource under `spec.template.metadata.annotations`.

== Troubleshooting Jaeger agent sidecar injection

Check to see if automatic injection is enabled in the `Deployment` resource for your application. If automatic injection for the Jaeger agent is enabled, there should be a `sidecar.jaegertracing.io/inject:"true"` annotation in the `Deployment` resource.
```

Reply:
This may be a caching issue?  I've already made this change.